### DISCUSSION TITLE
Item Fx (TakeFx) doesn't map on Realearn

### DISCUSSION THREAD

**Discussion by LudovicoDalPonte** [helgoboss/helgobox#1326] [General]
Hello,

I'm having issues on using ReaLearn for Item Fx using a Behringer X Touch Mini.
For Mastering purposes I have all the songs in the same track and i apply Fx on the single clips using Item Fx (or Take Fx).
I'd like to map the istances of my Main Eq through Realearn but it doesn't learn the parameters when i move them. It works fine when I try to map the Fx on normal tracks.
I put Realearn on the main track because putting it to Item Fx leads to an error.
Anybody have the same issue? 

Best regards, 

Ludovico

**Comment by helgoboss** [comment]
Modifying item FX params is not supported currently

**Reply by LudovicoDalPonte** [reply]
Thank You Helgoboss!
